werkzeug/http.py

```diff
@@ -374,7 +374,7 @@ def parse_options_header(value, multiple=False):
             return tuple(result)
         value = rest
 
-    return tuple(result)
+    return tuple(result) if result else ('', {})
```

Comment:
Anyway I just wanted to point out that one of those places could cover both
usecases. It seems there is a shortcut intended to minimize computing effort
for such values. I think that something like `if not value.strip(' ,')` might
work, but that depends on whether there is other input that leads to `result`
being empty.

On Thu, Aug 25, 2016 at 06:39:50AM -0700, Lars Holm Nielsen wrote:
> > @@ -374,7 +374,7 @@ def parse_options_header(value, multiple=False):
> >              return tuple(result)
> >          value = rest
> >  
> > -    return tuple(result)
> > +    return tuple(result) if result else ('', {})
> 
> Thanks. It also has to protect against values such as e.g. `` , `` so not sure if fully possible to move up in the beginning of the function but I'll give it a try.
> 
> -- 
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/pallets/werkzeug/pull/996/files/58627a4bcec0d9b294eef0e2e1e8bfc84d831361#r76244002